index.html

```diff
@@ -135,13 +135,13 @@ <h1>Tonic</h1>
                      </ul>
                  </div>
                  <div class="works">
-                     <a href="#"><p>See Project</p></a>>
+                     <a href="#"><p>See Project</p></a>
                  </div>
             </div>  
         </div>
         <div class="portfolio">
-            <div class="snapshot">
-                <img src="./images/portfolio3.png" alt="fourth portfolio">
+            <div class="snapshot4">
+                <!-- <img src="./images/portfolio3.png" alt="fourth portfolio"> -->
```

Comment:
[OPTIONAL] - Please clean all commented code in your HTML file to be your code clean and professional.